docs/41-managing-storage-via-R.md

```diff
@@ -0,0 +1,35 @@
+# Managing blob files in Azure Storage
+## Accessing your storage files through R
+Without installing Azure Storage Explorer or using the Azure Portal, users can access their resources through doAzureParallel wrapper functions around rAzureBatch's API calls.
+
+A container provides a grouping of a set of blobs. All blobs must be in a container. An account can contain an unlimited number of containers. A container can store an unlimited number of blobs. **Note** that the container name must be lowercase.
```

Comment:
**Note** that the container name must be lowercase.

-> Point them to the Azure storage naming requirements: https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata

Reply:
Okay will replace